app/controllers/home_controller.rb

```diff
@@ -53,11 +53,12 @@ def index
     @user_count = User.count
     @work_count = Work.posted.count
     @fandom_count = Fandom.canonical.count
-    @admin_posts = AdminPost.non_translated.find(:all, :order => "created_at DESC", :limit => 3)
-    @admin_post_show_more = AdminPost.count > 3
-    render :action => "index", :layout => "home"
-  end
 
+    @homepage = Homepage.new(@current_user)
+
+    @hide_dashboard = true
+    render action: 'index', layout: 'application'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.